### DISCUSSION TITLE
Module '"@tanstack/router"' has no exported member 'Outlet'

### DISCUSSION THREAD

**Discussion by ferrybig** [TanStack/router#558] [General]
I'm getting the following error when trying to use the router:

    Module '"@tanstack/router"' has no exported member 'Outlet'

I'm having the following versions installed in my package.json:

      "@tanstack/router": "0.0.1-beta.83",
      "@tanstack/router-devtools": "0.0.1-beta.83",

These version numbers are _exactly the same as on the following example_: https://stackblitz.com/github/tanstack/router/tree/beta/examples/react/with-react-query?embed=1&theme=light which I expect to work (the live preview does not support firefox, so i cannot see it working)

I found a related discussion here: https://github.com/TanStack/router/issues/547, which says to install `@tanstack/react-router`, but NPM says that the matching version was not found:

```
$ npm i @tanstack/react-router
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @tanstack/router-core@0.0.1-beta.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

What am I missing?

**Comment by livevsonline** [comment]
Try this one :) latest pre-release :)

"@tanstack/react-router": "^0.0.1-beta.83",

**Reply by cjconaty5** [reply]
Thanks, that worked for me.  I got this cmd `npm install @tanstack/router@beta` from [docs](https://tanstack.com/router/v1/docs/adapters/react-router), should we be using `npm install @tanstack/react-router@beta` instead?
